### PR TITLE
pull `AccessibleBuffer` out of `AccessibilityManager`

### PR DESCRIPTION
**Pull request by meganrogge**
Not sure where the code that makes `Shift Tab` enter this mode is, but it doesn't work when not using screen reader mode. Perhaps that's fine though and will leave it up to the embedders

### CODE REVIEW

File: src/browser/Terminal.ts [hunk 1]
```diff
@@ -576,6 +578,8 @@ export class Terminal extends CoreTerminal implements ITerminal {
     // Listen for mouse events and translate
     // them into terminal mouse protocols.
     this.bindMouse();
+
```
Comment (meganrogge) [inline src/browser/Terminal.ts:581]: I had to put this here so that dimensions from the render service wouldn't be 0. lmk if there's a better place

Comment (Tyriar) [reply src/browser/Terminal.ts:581]: This is the right function, I'd put it just below the `AccessibilityManager`s init to be a little more organized. If you needed to move it here then something must not have been migrated to the new file correctly as it was working before when bundled into `AccessibilityManager`.

Unless that only worked when toggling screenReaderMode on after Terminal was `open`ed? In that case there's probably some other event we're not listening to?

Comment (meganrogge) [reply src/browser/Terminal.ts:581]: that's where I put it originally and it didn't work, so seems like we're missing something

Comment (meganrogge) [reply src/browser/Terminal.ts:581]: it needs to be after this to work

```
    this._charSizeService.measure();
```

**Pull request merged**